### PR TITLE
fix: dependency versions

### PR DESCRIPTION
**Pull request by bruno-garcia**
As it was pointed out on #5, it's not possible to run the code with the latest tooling.

This PR updates gradle, sentry and test packages.
I've tested on Android emulator (API 29) and iPhone 11 Pro simulator

### CODE REVIEW

File: android/gradle.properties [hunk 1]
```diff
@@ -1 +1,2 @@
 org.gradle.jvmargs=-Xmx1536M
+android.enableR8=true
```
Comment (marandaneto) [inline android/gradle.properties]: `android.enableR8` is enabled by default on AGP >= 3.4.x

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
     android:versionCode="1"
     android:versionName="0.0.1">
 
-    <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="21" />
+    <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="29" />
```
Comment (marandaneto) [inline android/app/src/main/AndroidManifest.xml]: what's about moving this line to the Gradle file? setting up `minSdkVersion` and `targetSdkVersion` right next to `applicationId`

File: android/build.gradle [hunk 3]
```diff
@@ -23,7 +25,3 @@ subprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
-
-task wrapper(type: Wrapper) {
-    gradleVersion = '2.14.1'
-}
```
Comment (marandaneto) [inline android/build.gradle]: as far as I can see, Gradle wrapper is not versioned, so you might need to keep these lines and `BUMP` to `5.6.4`

File: android/gradle.properties (hunk 1 above)
Comment (bruno-garcia) [reply android/gradle.properties]: IDE added automatically. But maybe before I bumped AGP. Will remove

File: android/build.gradle (hunk 3 above)
Comment (bruno-garcia) [reply android/build.gradle]: I changed the gradle wrapper but didn't push. I'll revert this then.

**Comment by bruno-garcia** [comment]
Thanks @marandaneto 
@yjbanov I hope this is what you're looking for

**Comment by yjbanov** [comment]
> Thanks @marandaneto
> @yjbanov I hope this is what you're looking for

I wasn't actively looking for anything lately, as I haven't been actively maintaining this repo :)

The change looks reasonable to me. I'll test it and merge. Thanks!

**Comment by bruno-garcia** [comment]
Thanks!

@yjbanov if I add CI with travis-ci on my fork and PR back the configuration, is this something interesting to you? I notice travis.org is what you use for flutter/sentry

**Comment by yjbanov** [comment]
> @yjbanov if I add CI with travis-ci on my fork and PR back the configuration, is this something interesting to you? I notice travis.org is what you use for flutter/sentry

Yeah, that would be fantastic!

**Comment by bruno-garcia** [comment]
#7 includes these changes and also .travis.yml to build the iOS and Android apps

**Pull request closed without merging**